src/content/en/updates/2017/12/chrome-64-deprecations.md

```diff
@@ -0,0 +1,74 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 64 to help you plan. In this version, performance API changes, removal of support for multiple shadow roots, and removal of a WebKit API.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: deprecations,removals,chrome64 #}
+{# wf_blink_components: Blink>PerformanceAPIs,Blink>DOM,Blink>CSS #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 64 to help you plan. In this version, performance API changes, removal of support for multiple shadow roots, and removal of a WebKit API. #}
+
+# Deprecations and removals in Chrome 64 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes some of the deprecations and removals in Chrome
+64, which is in beta as of December 14. Visit the
+[deprecations page](/web/updates/tags/deprecations) for more deprecations and
+removals from this and previous versions of Chrome. This list is subject to
+change at any time.
+
+## Deprecate chrome.loadTimes()
+
+The `chrome.loadTimes()` method is a non-standard API that enables collecting
+loading-related metrics in real-world operations. A [Resource Timing Level
+1](https://www.w3.org/TR/resource-timing-1/) API has existed for several years
+while the current version of it is a W3C Candidate Recommendation. Chrome itself
+has been working to implement these for some time. For these reasons,
+the `chrome.loadTimes()` method is deprecated in Chrome 64. Removal is
+anticipated some time in late 2018.
+
+Developers needing an alternative should look at several spec-based APIs. Most
```

Comment:
https://github.com/google/WebFundamentals/pull/5500 should be published by the time this goes out, so it probably makes more sense to just link to there (rather than include this information).